### PR TITLE
Using govuk frontend styling for in IOO service start page

### PR DESCRIPTION
**Pull request by stuart-madetech**
Using GOV UK GDS frontend toolkit in international online offer. 

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/ioo-400
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [x] Includes screenshot(s) - ideally before and after, but at least after
- Before:
Screenshot 2023-02-13 at 08 34 37

- After:
Screenshot 2023-02-13 at 08 32 27

### Housekeeping

- [x] Ran the `make manage download_geolocation_data` command
- [x] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [ ] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: international_online_offer/templates/ioo/includes/phase_banner.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="govuk-width-container">
+    <div class="govuk-phase-banner">
+        <p class="govuk-phase-banner__content">
+        <strong class="govuk-tag govuk-phase-banner__content__tag atlas-body">
+            beta
+        </strong>
+        <span class="govuk-phase-banner__text atlas-body">
+            This is a new service – your <a class="govuk-link atlas-body" href="#">feedback</a> will help us to improve it.
+        </span>
```
Comment (mforner13) [inline international_online_offer/templates/ioo/includes/phase_banner.html:9]: Do we know where the feedback link will lead to?

File: international_online_offer/templates/ioo/index.html [hunk 2]
```diff
@@ -9,17 +9,51 @@
...
+                <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+              </svg></a>
+          </div>
+        </div>
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <h3 class="govuk-heading-l govuk-!-margin-top-9 atlas-h--m">Personal Data</h3>
+            <p class="govuk-body atlas-body">
+              We’ll never share your data with third parties. View our website’s terms and conditions
+            </p>
```
Comment (mforner13) [inline international_online_offer/templates/ioo/index.html:54]: Should the 'terms and conditions' text be a link to our terms and conditions page?

**Review by mforner13 — CHANGES_REQUESTED** [review]
I see you've brought in the gov uk style kit as an npm package. It looks like we've already got the main css file [here](https://github.com/uktrade/great-cms/blob/develop/core/components/static/stylesheets/govuk-elements-styles.css). I think it's best to not duplicate it, so it might be worth trying to remove that css file and see if just the npm package can be used?

File: international_online_offer/templates/ioo/index.html (hunk 2 above)
Comment (stuart-madetech) [reply international_online_offer/templates/ioo/index.html:55]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +17,7 @@
 * KLS-266 - Get survey details from directory API
 * KLS-203 - Upgrade django and Wagtail
 * IOO-399 - International Online Offer project setup
+* IOO-400 - International Online Offer adding govuk frontend toolkit and using in index
```
Comment (mforner13) [inline CHANGELOG.md:20]: Might be worth updating this now we're not adding the whole toolkit :)

File: international_online_offer/sass/_inset-text.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
+@import '../../sass/partials/colours';
+
+
+.inset-text {
+  padding: 15px;
+  // Margin top intended to collapse
+  // This adds an additional 10px to the paragraph above
+  // @include govuk-responsive-margin(6, "top");
+  // @include govuk-responsive-margin(6, "bottom");
```
Comment (mforner13) [inline international_online_offer/sass/_inset-text.scss:9]: Should these be uncommented? :)

Comment (stuart-madetech) [reply international_online_offer/sass/_inset-text.scss]: Done :)

**Review by mforner13 — APPROVED** [review]
👍

**Pull request merged**